### PR TITLE
Update README to describe the application

### PR DESCRIPTION
**Pull request by JackWilb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,14 @@
-# vue-nodelink
+# Node-link View
 
-## Project setup
-```
-yarn install
-```
+This is the node-link data visualization app for the Multinet project. It is a typescript application, written using the Vue.js framework, that allows users to visualize network data hosted on a Multinet instance.
```
Comment (waxlamp) [inline README.md]: Wrap at 80.

File: README.md [hunk 2]
```diff
@@ -1,29 +1,14 @@
...
+To get the application up and running in dev mode:
 
-### Run your tests
-```
-yarn run test
-```
+1. Ensure that you have Node and yarn installed.
+1. Check out this repository and move into it (e.g., `cd view-nodelink`).
+1. Install the Yarn dependencies: `yarn install`.
+1. Run the application: `yarn serve`.
```
Comment (waxlamp) [inline README.md]: Use ascending numbers.

File: README.md [hunk 3]
```diff
@@ -1,29 +1,14 @@
...
 
-### Lints and fixes files
-```
-yarn run lint
-```
-
-### Customize configuration
-See [Configuration Reference](https://cli.vuejs.org/config/).
+For further details, including how to set up a full multinet instance, please see the [full documentation](https://multinet-app.readthedocs.io).
```
Comment (waxlamp) [inline README.md]: Wrap at 80.

**Review by waxlamp — DISMISSED** [review]
Approved, with similar caveats at the adjacency matrix README PR.

File: README.md (hunk 3 above)
Comment (JackWilb) [reply README.md]: See 2ab71f4

File: README.md (hunk 1 above)
Comment (JackWilb) [reply README.md]: See 2ab71f4

File: README.md (hunk 2 above)
Comment (JackWilb) [reply README.md]: See 2ab71f4

**Review by waxlamp — APPROVED** [review]

**Pull request merged**